goserver/server/server.go

```diff
@@ -0,0 +1,30 @@
+package server
+
+import (
+	"net/http"
+)
+
+const (
+	launchPath = "/launch"
+)
+
+// Server handles incoming HTTP requests.
+type Server struct {
+	AuthorizedFHIREndPoint string
+}
+
+// Run starts HTTP server on port 8080
+func (s *Server) Run() {
+	http.HandleFunc(launchPath, s.handleLaunch)
+
+	http.ListenAndServe(":8080", nil)
```

Comment:
The port should be configurable via a flag.

Reply:
Done.